src/lib/components/pages/anime_info/error.svelte

```diff
@@ -62,13 +64,12 @@
             </a>
         </error-context>
         <character-image
-            class="pointer-events-none relative flex items-end justify-center md:w-[45dvw]"
+            class:order-1={on_left}
+            class:order-2={on_right}
```

Comment:
This is what you had to do

Reply:
Okay I did same, but I dont think we need this much `order` classes
just one add `order-1` for `on_right`
I mean no need this `order-` `order-2` ... on both `charater-image` and `error-context`

Reply:
>I mean no need this order- order-2 ... on both charater-image and error-context

better to separate logics than to deal with snakes later